### ISSUE TITLE
Loss Weight ablation experiment

### ISSUE THREAD

**Issue by JinYu1998** [huggingface/distil-whisper#75]
Have you tried the effect of different loss weights on the distillation results ?

**Comment by sanchit-gandhi** [comment]
Hey @JinYu1998 - we did a coarse sweep over the KL weights when setting up preliminary experiments on just the LibriSpeech corpus. We found the setting from DistilBART to be best, and so committed to this for the rest of the project. We didn't do any further tuning of the loss weights on our full training set. You can find an ablation over the loss terms (not weights) in page 26 of [the paper](https://arxiv.org/pdf/2311.00430.pdf).

**Comment by JinYu1998** [comment]
> Hey @JinYu1998 - we did a coarse sweep over the KL weights when setting up preliminary experiments on just the LibriSpeech corpus. We found the setting from DistilBART to be best, and so committed to this for the rest of the project. We didn't do any further tuning of the loss weights on our full training set. You can find an ablation over the loss terms (not weights) in page 26 of [the paper](https://arxiv.org/pdf/2311.00430.pdf).

Thank you for your reply, I have previously worked on dynamic temperature distillation on classifieds, and just recently finished this work. I'm very interested in distillation in whisper, and look forward to combining my work with distill whisper very well.

**Comment by abdulmominseo** [comment]
My 52-Week Weight Loss Journey Unveiled | Ian Melton's weight loss story
Rating-⭐⭐⭐⭐⭐
https://youtu.be/KTS0AXO3YHY?si=mdUW6BxiTfGWeibd

**Comment by abdulmominseo** [comment]
Delicious & Exciting Diet Foods : Weight Loss Food full video - https://youtu.be/4Kr8gtd2oss?si=1HVNwuBNTKgr4XCL
